src/components/common/Directors.vue

```diff
@@ -452,27 +455,44 @@
                   -->
 
                   <div class="form__row form__btns">
-                    <v-btn color="error"
-                      class="remove-edit-btn"
-                      v-show="isNew(director)"
-                      @click="deleteDirector(director.id)"
-                    >
-                      <span>Remove</span>
-                    </v-btn>
-                    <v-btn color="error"
-                     class="reset-edit-btn"
-                     v-show="!isNew(director) && editFormShowHide.showName"
-                     @click="restoreDirName(director.id, true)"
-                     :disabled="!isNameChanged(director)"
-                    >
-                      <span>Reset</span>
-                    </v-btn>
+                    <!-- v-show doesn't support <template> so use a <div> instead -->
+                    <div v-show="isNew(director)">
+                      <v-btn color="error" outlined
+                        class="remove-edit-btn"
+                        @click="deleteDirector(director.id)"
+                      >
+                        <span>Remove</span>
+                      </v-btn>
+                    </div>
+
+                    <!-- v-show doesn't support <template> so use a <div> instead -->
+                    <div v-show="!isNew(director)">
+                      <!-- FUTURE: implement this -->
+                      <v-btn color="error" outlined
```

Comment:
I added this button to be consistent with change of director name (but I left it disabled as implementing the functionality is beyond the scope of these tickets).